tempest/api/workloadmgr/multiattach_volumes/test_multiattach_volumes.py

```diff
@@ -515,16 +530,26 @@ def test_01_multiattach_volumes(self):
 
             payload = self.create_restore_json(rest_details)
             # Trigger selective restore of full snapshot
-            self._selective_restore(payload, [fip[2], fip[3]], md5sums_before_full)
+            self._selective_restore(rest_details, [fip[2], fip[3]], md5sums_before_full)
 
             # Trigger selective restore of incremental snapshot
-            self._selective_restore(payload, [fip[4], fip[5]], md5sums_before_incr,
+            self._selective_restore(rest_details, [fip[4], fip[5]], md5sums_before_incr,
                                     False)
             reporting.test_case_to_write()
             tests[5][1] = 1
 
-            # Inplace restore
+            # Selective restore with different volume mapping
             reporting.add_test_script(tests[6][0])
+            rest_details['volume_type'] = CONF.volume.volume_type
```

Comment:
What is the difference in actual volume type and the new volume type in case of this test case ?

Reply:
actual volume type is multiattach and this is the first volume type set in var files which would be non multiattch.